### PR TITLE
INI settings for 'Hasbro Family Game Night 2'

### PR DESCRIPTION
**Pull request by jack-1313**
Fixes :
https://bugs.dolphin-emu.org/issues/12534

### CODE REVIEW

**Comment by Rumi-Larry** [comment]
Cool, but the pr title should have the name of the game :)

**Comment by mbc07** [comment]
If Sync GPU Thread can fix this, I think it should be preferred over disabling Dual Core. As bad as Dual Core is, it can still provide nice speed boosts especially for people with low end machines...

**Comment by jack-1313** [comment]
> Cool, but the pr title should have the name of the game :)

I hope it is fine now.
 (.❛ ᴗ ❛.)

**Comment by JMC47** [comment]
I saw conflicting reports that syncGPU may not always fix it when on the unofficial discord.

**Comment by jack-1313** [comment]
> If Sync GPU Thread can fix this, I think it should be preferred over disabling Dual Core. As bad as Dual Core is, it can still provide nice speed boosts especially for people with low end machines...

So, I tested using “Sync GPU Thread” enabled, and this fixed this issue. _Although, it is not much faster than single core as I wasn’t able to notice any significance difference._

But still I have made the required changes in the file.
Hopefully, everything is ok now.

**Comment by jack-1313** [comment]
> I saw conflicting reports that syncGPU may not always fix it when on the unofficial discord.

For me both (Disable Dual Core and Sync GPU Thread) worked, it’s now up to you guys to consider which option would be the best. Maybe someone else should test it as well. 

@JMC47 
When did you get that feedback? If it was over one month ago, then it could be me, as I had talked to you before filing the bug.

**Comment by Rumi-Larry** [comment]
> 
> 
> > Cool, but the pr title should have the name of the game :)
> 
> I hope it is fine now.
> (.❛ ᴗ ❛.)

I suggest "INI settings for 'Hasbro Family Game Night 2'"

File: Data/Sys/GameSettings/R6X.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# R6XP69, R6XE69 - Hasbro Family Game Night 2
+
+[Core]
+ # Dual Core mode hangs after the intro.       
+ SyncGPU = True
+
+[Video_Settings]
+ # The intro logo shutters.
```
Comment (AdmiralCurtiss) [inline Data/Sys/GameSettings/R6X.ini:8]: I assume you mean 'stutters', not 'shutters'.

**Comment by jack-1313** [comment]
I have corrected the spelling mistake, and **for now, I have reverted back to "Disable Dual Core"**, because of this:-

> I saw conflicting reports that syncGPU may not always fix it when on the unofficial discord.

**Comment by jack-1313** [comment]
Can this PR be merged? 

“Disable Dual Core” and “Safe Texture Cache Accuracy” completely solve the issue.

As for “Sync GPU”, we don’t know if it is able to fix the hang for all devices. Until we get more details, this GameINI seems good to me. And if we later get the confirmation that “Sync GPU” fixes, then I will make a new PR for the required changes.

_Also, I tried my best to merge all my 4 commits, but since I am new to GitHub, I wasn’t able to do so. Hopefully you guys can use the “Squash and Merge” option._

**Comment by Rumi-Larry** [comment]
The title is good, but you missed the second single quote

**Comment by jack-1313** [comment]
> The title is good, but you missed the second single quote

Oh, sorry for that. My first PR has truly turned to be a disaster with so many mistakes. 

Hope everything is now good to go.

**Comment by mbc07** [comment]
A simple change like this is very unlikely to get merged until you squash your commits...

**Review by jack-1313 — COMMENTED** [review]
I think this is the squashed one, the only which needs to be merged?

I am confused, have I done anything wrong?

**Comment by JosJuice** [comment]
It looks like you first created a squashed commit, and then merged/pulled origin. You need to do it without merging/pulling origin. If the reason why you merged/pulled origin is because git push wouldn't let you push otherwise, you need to use the `-f` option to `git push` to override that.

**Comment by jack-1313** [comment]
When I try squashing all the 6 commits, I get this error-
"Unable to squash. Squashing replays all commits up to the last one required for the squash. A merge commit cannot exist among those commits."

Do I need commit all 6, or only the 3 that I had earlier tried to squash?

**Comment by JosJuice** [comment]
Try `git reset --hard 8f107191cbed3d993a2db41440e24c77449491cd`

This will bring you back to how things were right after making the 8f107191cbed3d993a2db41440e24c77449491cd commit, before making the merge commit.

**Comment by jack-1313** [comment]
@JosJuice 

When I try to `git push origin`, it asks for `git pull`, so I do that also.

After that it shows this message
"Please enter a commit message to explain why this merge is necessary, especially if it merges an updated upstream into a topic branch.
Lines starting with '#' will be ignored, and an empty message aborts the commit."

I am stuck here, can you please help what to do now?

**Comment by JosJuice** [comment]
Like I mentioned before, you shouldn't use `git pull`. Instead, use `git push -f` in place of `git push`.

**Comment by jack-1313** [comment]
So, finally, somehow I am able to squash all my commits into one with JosJuice's help.
Let me know if there is something else that also needs to be done.

**Comment by JMC47** [comment]
@dolphin-emu-bot rebuild

**Comment by JMC47** [comment]
I double checked single core vs dualcore + syncGPU performance in a few games just in case, and it doesn't seem to be a major difference.  If anything, the safe texture cache is a harder hit, but it appears to be necessary to put it to safe.  Seems fine to me and the game isn't really playable in master anyway unless you use single core.

**Pull request merged**